bool dio0
bool dio1
bool dio2
bool dio3
bool dio4
bool dio5
bool dio6
bool dio7
bool dio8
bool dio9
bool dio10
bool dio11
bool dio12
bool dio13
bool dio14
bool dio15
bool dio16
bool dio17
bool dio18
bool dio19
bool dio20
bool dio21
bool dio22
